tailer.go

```diff
@@ -83,8 +84,9 @@ func maximumNameLength(filenames []string) int {
 	max := 0
 	for _, name := range filenames {
 		base := filepath.Base(name)
-		if len(base) > max {
-			max = len(base)
+		width := runewidth.StringWidth(base)
```

Comment:
As this is a bit more costly than len(string), calling only once per loop and instead using temporary variable.